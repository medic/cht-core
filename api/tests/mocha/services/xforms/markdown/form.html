<root xmlns:xf="http://www.w3.org/2002/xforms" xmlns:orx="http://openrosa.org/xforms" xmlns:enk="http://enketo.org/xforms" xmlns:odk="http://www.opendatakit.org/xforms" xmlns:kb="http://kobotoolbox.org/xforms" xmlns:esri="http://esri.com/xforms" xmlns:oc="http://openclinica.org/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa">
  <form autocomplete="off" novalidate="novalidate" class="or clearfix pages" dir="ltr" id="note_demo">
<!--This form was created by transforming a OpenRosa-flavored (X)Form using an XSL stylesheet created by Enketo LLC.--><section class="form-logo"> </section><h3 dir="auto" id="form-title">Note Demo</h3><select id="form-languages" style="display:none;" data-default-lang="en"><option value="en">en</option> </select>
  
  
    <section class="or-group-data or-appearance-field-list or-appearance-summary " name="/notedata"><label class="question non-select "><span lang="" class="question-label active">Name</span><input type="text" name="/notedata/name" data-type-xml="string"/></label><label class="question non-select "><span lang="en" class="question-label active" data-itext-id="name_note:label">
# Hello <span class="or-output" data-value="/notedata/name"> </span>!
</span><input type="text" name="/notedata/name_note" data-type-xml="string" readonly="readonly"/></label><label class="question non-select or-appearance-h2 or-appearance-yellow or-appearance-markdown "><span lang="en" class="question-label active" data-itext-id="lmp_note:label"><span class="or-output" data-value="/notedata/name"> </span> is currently &lt;span style="font-family:monospace"&gt;12 weeks&lt;/span&gt; pregnant.</span><input type="text" name="/notedata/lmp_note" data-type-xml="string" readonly="readonly"/></label><label class="question non-select "><span lang="en" class="question-label active" data-itext-id="patient_details:label">Remind woman of the **6 STEPS TO A HEALTHY PREGNANCY:**

             &lt;ol style="margin-left:1em; list-style: decimal inside;"&gt;&lt;li&gt;Attend regular ANC visits&lt;/li&gt;&lt;li&gt;Sleep under a treated net **every** night&lt;/li&gt;&lt;li&gt;<span class="or-output" data-value="/notedata/name"> </span> is in her **first trimester**. Remind her to take supplements:&lt;ul style="margin-left:1em; list-style: disc inside;"&gt;&lt;li&gt;_Take Iron Folate daily_&lt;/li&gt;&lt;/ul&gt;&lt;/li&gt;&lt;li&gt;Eat well: Eat more often than usual and eat a variety of foods to give you strength and to help your baby grow&lt;/li&gt;&lt;li&gt;Deliver your child at a health center&lt;/li&gt;&lt;li&gt;Breastfeed the baby **immediately** after birth&lt;/ol&gt;</span><input type="text" name="/notedata/patient_details" data-type-xml="string" readonly="readonly"/></label><label class="question non-select "><span lang="en" class="question-label active" data-itext-id="referrals:label">Referrals&lt;I class="fa fa-hospital-o"&gt;&lt;/i&gt;</span><input type="text" name="/notedata/referrals" data-type-xml="string" readonly="readonly"/></label><label class="question non-select "><span lang="en" class="question-label active" data-itext-id="submit_note:label">&lt;h4 style="text-align:center;"&gt;Click the Submit button at the bottom of the form.&lt;/h4&gt;</span><input type="text" name="/notedata/submit_note" data-type-xml="string" readonly="readonly"/></label>
            </section><!--end of group -->
  
</form>
</root>