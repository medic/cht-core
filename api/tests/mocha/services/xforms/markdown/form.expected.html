<form autocomplete="off" novalidate="novalidate" class="or clearfix pages" dir="ltr" id="note_demo">
<section class="form-logo"> </section><h3 dir="auto" id="form-title">Note Demo</h3><select id="form-languages" style="display:none;" data-default-lang="en"><option value="en">en</option> </select>
  
  
    <section class="or-group-data or-appearance-field-list or-appearance-summary " name="/notedata"><label class="question non-select "><span lang="" class="question-label active">Name</span><input type="text" name="/notedata/name" data-type-xml="string"></label><label class="question non-select "><span lang="en" class="question-label active" data-itext-id="name_note:label"><h1>Hello <span class="or-output" data-value="/notedata/name"> </span>!</h1></span><input type="text" name="/notedata/name_note" data-type-xml="string" readonly="readonly"></label><label class="question non-select or-appearance-h2 or-appearance-yellow or-appearance-markdown "><span lang="en" class="question-label active" data-itext-id="lmp_note:label"><span class="or-output" data-value="/notedata/name"> </span> is currently <span style="font-family:monospace">12 weeks</span> pregnant.</span><input type="text" name="/notedata/lmp_note" data-type-xml="string" readonly="readonly"></label><label class="question non-select "><span lang="en" class="question-label active" data-itext-id="patient_details:label"><p>Remind woman of the <strong>6 STEPS TO A HEALTHY PREGNANCY:</strong></p>             <ol style="margin-left:1em; list-style: decimal inside;"><li>Attend regular ANC visits</li><li>Sleep under a treated net <strong>every</strong> night</li><li><span class="or-output" data-value="/notedata/name"> </span> is in her <strong>first trimester</strong>. Remind her to take supplements:<ul style="margin-left:1em; list-style: disc inside;"><li><em>Take Iron Folate daily</em></li></ul></li><li>Eat well: Eat more often than usual and eat a variety of foods to give you strength and to help your baby grow</li><li>Deliver your child at a health center</li><li>Breastfeed the baby <strong>immediately</strong> after birth</ol></span><input type="text" name="/notedata/patient_details" data-type-xml="string" readonly="readonly"></label><label class="question non-select "><span lang="en" class="question-label active" data-itext-id="referrals:label">Referrals<I class="fa fa-hospital-o"></i></span><input type="text" name="/notedata/referrals" data-type-xml="string" readonly="readonly"></label><label class="question non-select "><span lang="en" class="question-label active" data-itext-id="submit_note:label"><h4 style="text-align:center;">Click the Submit button at the bottom of the form.</h4></span><input type="text" name="/notedata/submit_note" data-type-xml="string" readonly="readonly"></label>
            </section>
  
</form>
